### ISSUE TITLE
track defined as "was already downloaded" but actually not

### ISSUE THREAD

**Issue by pumpkinknight** [vitiko98/qobuz-dl#110]
hey guys, i tried to use the tool on the following album but got this issue, any one got the similar case?

https://open.qobuz.com/album/0002894152962

![001.png](https://i.loli.net/2021/06/19/FP3zeMtOiyZdCvs.png)

**Comment by chrishunt** [comment]
This may help - from [the README](https://github.com/vitiko98/qobuz-dl/blob/95d32f99b1d3c663d142125816b0c2915e3ba1a9/README.md#other):

> By default, qobuz-dl will skip already downloaded items by ID with the message This release ID ({item_id}) was already downloaded. To avoid this check, add the flag --no-db at the end of a command. In extreme cases (e.g. lost collection), you can run qobuz-dl -p to completely reset the database.

**Comment by pumpkinknight** [comment]
@chrishunt Thank you for attention! Sorry for missing the command line in the snapshot, I usually put --no-db for all downloads, here's the re-try snopshot 
![001](https://user-images.githubusercontent.com/29995848/123429728-da3dfa80-d5f9-11eb-90fd-0c7d1b921b4a.JPG)

**Comment by sramsay100** [comment]
When using --no-db, the sscript reverts to checking for Artist/folder_format/track_format existing on the in your default_folder location. If it exists, you will also get the message you are experiencing.

If you want to download this again, for change in quality etc. then you will need to delete the files/directory that contains this ablum first.

**Comment by pumpkinknight** [comment]
> 

Hi, thank you for replying the message. To re-produce or verify this issue I tried several times to download **_this_** album with default_folder defintely empty. I ven though maybe that's caused by too long folder name or similar reason.

https://open.qobuz.com/album/0002894152962

**Comment by sramsay100** [comment]
Yeah, I cannot download that one either. Definitely a bug, caused by foriegn characters In the artist name I think.
